coeus-impl/src/main/java/org/kuali/coeus/sys/framework/workflow/KcPostProcessor.java

```diff
@@ -31,4 +33,15 @@ public ProcessDocReport doRouteStatusChange(DocumentRouteStatusChange statusChan
         return ((PostProcessorService) KcServiceLocator.getService("kcPostProcessorService")).doRouteStatusChange(statusChangeEvent);
     }
 
+    @Override
+    public ProcessDocReport doRouteLevelChange(final DocumentRouteLevelChange levelChangeEvent) throws Exception {
+        return ((PostProcessorService) KcServiceLocator.getService("kcPostProcessorService")).doRouteLevelChange(levelChangeEvent);
```

Comment:
Interesting that these methods were never implemented before.... Seems like a gap